docs/manual/developer/06_contributing_with_content.md

```diff
@@ -274,7 +274,7 @@ A rule may contain those reference-type attributes:
     </thead>
     <tbody>
     <tr class="odd">
-    <td><p>cis</p></td>
+    <td><p>cis&lt;product&gt;&lt;product_version&gt;</p></td>
```

Comment:
Missing an `@` like `stigid` below :)

And thanks for the docs updates!

Reply:
thanks for the catch